packages/OreIdReact/src/components/Observables/Observables.tsx

```diff
@@ -0,0 +1,60 @@
+import isEqual from "lodash/isEqual";
```

Comment:
I'm assuming you checked but is there an equivalent function in the shared library?

Reply:
@deckb I haven't found it, but even if it has I'd rather not use it. Our shared lib is insanely big and importing this way webpack only uses this piece of code from lodash (resulting in few lines of code).

Reply:
@yardz there is not a function in our shared library - but I created one in oreid-js Helpers called objectsAreEqual which does the same thing e.g. Helpers.objectsAreEqual(objA, objB). If we are using lodash in very few places, it would be best to remove if we dont need it